src/main/java/org/synyx/urlaubsverwaltung/sicknote/sicknote/SickNoteMailService.java

```diff
@@ -92,6 +95,21 @@ void sendEndOfSickPayNotification() {
         }
     }
 
+    /**
+     * Sends information about a created sick note to the applicant
+     *
+     * @param sickNote that has been created
+     */
+    @Async
+    void sendCreatedToApplicant(SickNote sickNote) {
```

Comment:
Is it really an applicant? Should be a little bit different wording here in contrast to absences?
Maybe `sendCreatedToSickPerson`?